### PR TITLE
Feature/145/about content on mobile

### PR DESCRIPTION
**Pull request by GabeDiniz**
✨ **Issue:** https://github.com/LaurierHawkHacks/Landing/issues/145
🔀 **Branch:** feature/145/about-content-on-mobile

**✏ What was changed?**
- Update SVG for mobile
- Change text color to match Hero section (hopefully its more readable)

Let me know what you guys think about the text color. It matches the Hero section now and I think its a little more readable. I don't know how I feel about the About Title and Content being different colors though. 

See the last screenshot to see the possible alternative for text-color. 

**📜 Screenshots**
Width: 390px
image

Width: 650px
image

Possible alternative?
image

### CODE REVIEW

**Comment by GabeDiniz** [comment]
Looks better with the gradient:
image
image

File: src/assets/hero-about/hero-about-desktop-bak.svg
Comment (teoh4770) [inline src/assets/hero-about/hero-about-desktop-bak.svg:1]: the naming of the image is weird? Can you change it?

...bak.svg?

**Review by teoh4770 — COMMENTED** [review]
Got one question: Why do we have repeated images in the assets/hero-about?

Screenshot 2024-03-24 at 6 12 34 PM

**Comment by teoh4770** [comment]
and I feel like **the about section paragraphs' color can be darker to make the text more obvious**.
I have run the color choices in the web accessibility color contrast checker, and it suggests that as well.

**Comment by teoh4770** [comment]
Screenshot 2024-03-24 at 6 31 57 PM

I also find that the spacing of the content is weird in desktop view.
I know it is not under your scope of your pr, we can make an issue for that after we have addressed this one first.

**Comment by teoh4770** [comment]
but besides that, I think it is better!

**Comment by GabeDiniz** [comment]
> Got one question: Why do we have repeated images in the assets/hero-about?
> 
> Screenshot 2024-03-24 at 6 12 34 PM

I was gonna remove that :) just waiting to see if everyone was happy with the final design. I didn't want to get rid of it until we all came to an agreement on the styling. Sorry about that, I should've said said that 😅

**Comment by GabeDiniz** [comment]
> and I feel like **the about section paragraphs' color can be darker to make the text more obvious**. I have run the color choices in the web accessibility color contrast checker, and it suggests that as well.

@teoh4770 do you have a color suggestion? What do you think of this (#1D4549)
image

**Comment by GabeDiniz** [comment]
> Screenshot 2024-03-24 at 6 31 57 PM
> 
> I also find that the spacing of the content is weird in desktop view. I know it is not under your scope of your pr, we can make an issue for that after we have addressed this one first.

You are right, its totally weird LOL.. I just fixed it. Should look a lot better now for all breakpoints.

**Comment by teoh4770** [comment]
> > and I feel like **the about section paragraphs' color can be darker to make the text more obvious**. I have run the color choices in the web accessibility color contrast checker, and it suggests that as well.
> 
> @teoh4770 do you have a color suggestion? What do you think of this (#1D4549)
> image
> 

As long as it's clear to see, and the colour is aligned with the title like the one you have. Others can suggest the colour as well

**Comment by teoh4770** [comment]
> > Got one question: Why do we have repeated images in the assets/hero-about?
> > 
> > Screenshot 2024-03-24 at 6 12 34 PM
> 
> I was gonna remove that :) just waiting to see if everyone was happy with the final design. I didn't want to get rid of it until we all came to an agreement on the styling. Sorry about that, I should've said said that 😅
We can always revert back to the change if we want to get back the original pic👍
And no worries though

**Comment by aidantrabs** [comment]
We'll talk about this tonight

**Pull request merged**